File: Upper case conversion - GFG/upper-case-conversion.java
//{ Driver Code Starts
//Initial Template for Java


import java.io.*;
import java.util.*;
class GfG
{
    public static void main(String args[])throws IOException
        {
            BufferedReader br = new BufferedReader( new InputStreamReader(System.in));
            int t = Integer.parseInt(br.readLine());
            while(t-->0)
                {
                    String str = br.readLine();
                    Solution ob = new Solution();
                    System.out.println(ob.transform(str));
                }
        }
}
// } Driver Code Ends


//User function Template for Java

class Solution
{
    public String transform(String s)
    {
        // code here
       String str[] = s.split("\\s");
       String res = "";
       for(String st : str){
           String firstChar = st.substring(0,1);
           String remaining = st.substring(1,st.length());
           res = res+firstChar.toUpperCase()+remaining+" ";
       }
       return res;
    }
}

File: Upper case conversion - GFG/upper-case-conversion.cpp
//{ Driver Code Starts
#include<bits/stdc++.h>
using namespace std;
string transform(string str);
int main()
{
    int t;
    cin>>t;
    cin.ignore();
    while(t--)
    {
        string str;
        getline(cin,str);
        cout<<transform(str)<<endl;
    }
    return 0;
}
// } Driver Code Ends



string transform(string s)
{
    // code here
      
        // s[0] = s[0]-32;
        // for(int i = 0; i<s.length(); i++){
        //     if(s[i]==' '){
        //         s[i+1] = s[i+1]-32;
        //     }
        // }
        // return s;
        
        // second method : 02
        
        s[0] = s[0]-'a'+'A';
        for(int i = 0; i<s.length(); i++){
            if(s[i]==' '){
                s[i+1] = s[i+1]-'a'+'A';
            }
        }
        return s;
}